database_files/hvac_minimum_requirement_boilers_IECC.json

```diff
@@ -0,0 +1,2050 @@
+[
+    {
+        "template": "90.1-2004",
```

Comment:
It seems that this is the ASHRAE data, not the IECC. All template names seem to include "90.1".

Reply:
Sorry, I forget to generate the IECC json...

Reply:
Upload the IECC json file.